File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/ResumeDTO.java
package com.example.befindingjob.dto;

import lombok.*;

import java.time.LocalDateTime;

@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ResumeDTO {
    private Integer resumeId;
    private Integer applicationId;
    private String resumeUrl;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/ResumeDTO.kt
package com.example.jobhub.dto

import java.time.LocalDateTime

data class ResumeDTO(
    val resumeId: Int? = null,
    val applicationId: Int,
    val resumeUrl: String,
    val createdAt: LocalDateTime = LocalDateTime.now(),
    val updatedAt: LocalDateTime? = null
)

